package.json

```diff
@@ -6,7 +6,6 @@
   "devDependencies": {
     "babel-cli": "6.8.0",
     "babel-plugin-lodash": "3.1.0",
-    "babel-plugin-rewire": "1.0.0-rc-2",
```

Comment:
I think you can just point this to my github repo instead of using the before_script approach. Both should work. NPM does a pretty good job of pulling in github repos.